tmt/steps/provision/testcloud.py

```diff
@@ -155,13 +158,67 @@ def import_testcloud() -> None:
 SSH_KEYGEN_TYPE = "ecdsa"
 
 DEFAULT_USER = 'root'
-DEFAULT_MEMORY = 2048          # MB
-DEFAULT_DISK = 40              # GB (maximum size allowed)
+DEFAULT_CPU_COUNT = 2
+DEFAULT_MEMORY: 'Size' = tmt.hardware.UNITS('2048 MB')
+DEFAULT_DISK: 'Size' = tmt.hardware.UNITS('40 GB')
 DEFAULT_IMAGE = 'fedora'
 DEFAULT_CONNECTION = 'session'
 DEFAULT_ARCH = platform.machine()
 
 
+def normalize_memory_size(
```

Comment:
If I'm not mistaken, the test `/tests/provision/virtual.testcloud` will fail because it provides disk and memory as type `int`. In the code, the `normalize_memory_size` and `normalize_disk_size` functions only allow types `None`, `pint.Quantity`, and `str`, so they throw a NormalizationError. Shouldn't the functions or the test be updated?

Reply:
The test does not provide disk and memory as integers - `--memory` is given *a sequence of characters* that *looks* like an integer. It would be successfully parsed into an integer, but we don't ask Click to do so. Click extracts the value from `sys.argv`, and after that, it might coerce the string into an integer if the option's `type` has been set to `int`. This isn't the case here, the value is delivered to our `normalize_memory_size()` untouched and unmodified, as a string.

Reply:
If the memory is specified by the `--memory` parameter, the value is indeed a string. However, if the memory is specified in `.fmf` file like this: `memory: 2048`, it will be an integer. In the test `/tests/provision/virtual.testcloud`, it is also specified in the `.fmf` file.

Reply:
Aha! Got it, good point.

Reply:
Fixed in 570a950a